src/main/java/mekanism/common/item/ItemAtomicDisassembler.java

```diff
@@ -60,41 +60,39 @@ public boolean canHarvestBlock(@Nonnull IBlockState state, ItemStack stack) {
     @Override
     public void addInformation(ItemStack itemstack, World world, List<String> list, ITooltipFlag flag) {
         super.addInformation(itemstack, world, list, flag);
-        list.add(LangUtils.localize("tooltip.mode") + ": " + EnumColor.INDIGO + getModeName(itemstack));
-        list.add(LangUtils.localize("tooltip.efficiency") + ": " + EnumColor.INDIGO + getEfficiency(itemstack));
+        Mode mode = getMode(itemstack);
+        list.add(LangUtils.localize("tooltip.mode") + ": " + EnumColor.INDIGO + mode.getModeName());
+        list.add(LangUtils.localize("tooltip.efficiency") + ": " + EnumColor.INDIGO + mode.getEfficiency());
     }
 
     @Override
     public boolean hitEntity(ItemStack itemstack, EntityLivingBase target, EntityLivingBase attacker) {
-        if (getEnergy(itemstack) > 0) {
-            if (attacker instanceof EntityPlayer) {
-                target.attackEntityFrom(DamageSource.causePlayerDamage((EntityPlayer) attacker), 20);
-            } else {
-                target.attackEntityFrom(DamageSource.causeMobDamage(attacker), 20);
-            }
-            setEnergy(itemstack, getEnergy(itemstack) - 2000);
+        double energy = getEnergy(itemstack);
+        int energyCost = MekanismConfig.current().general.DISASSEMBLER_WEAPON_USAGE.val();
+        int minDamage = MekanismConfig.current().general.DISASSEMBLER_DAMAGE_MIN.val();
+        int damageDifference = MekanismConfig.current().general.DISASSEMBLER_DAMAGE_MAX.val() - minDamage;
+        //If we don't have enough power use it at a reduced power level
+        double percent = energyCost == 0 ? 1 : energy / energyCost;
```

Comment:
`energy / energyCost` will be > 1 when it has plenty of charge, no?

Reply:
Good catch